### DISCUSSION TITLE
I’d like more height classes…

### DISCUSSION THREAD

**Discussion by eliotpalma** [tailwindlabs/tailwindcss#9012] [Ideas]
Hi,

I love tailwind css.

I don't think there are big enough height classes, for sections, the maximum height is h-96.

There is h-screen which is good, and i would't mind using vh but there's only one class of it.

I know that you can use padding on a section to give it's height but than percentage height classes of the children don't seem to work.

Thanks.

**Comment by cannap** [comment]
here you go https://tailwindcss.com/docs/height#arbitrary-values

**Reply by eliotpalma** [reply]
Hi,

I already knew about this, it is a way to achieve it but i'd still like more height classes.
Thank you for your comment though!
🌴
